graylog/Vagrantfile

```diff
@@ -4,46 +4,27 @@
 # Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
 VAGRANTFILE_API_VERSION = "2"
 
-Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  config.vm.box = "puphpet/debian75-x64"
-  config.vm.box_version = "2.0"
+Vagrant.configure(2)  do |config|
+  config.vm.box = "ubuntu/trusty64"
+  config.vm.hostname = "StepUp-log-Server"
```

Comment:
`Stepup-log-server` it'll be, the rest is out of scope for now. Will add tech debt issue.